test/test_distribution_generic.py

```diff
@@ -381,6 +388,7 @@ def __hash__(self):
         "dist.RelaxedBernoulli(temperature=case.temperature, logits=case.logits)",
         (("temperature", f"rand({batch_shape})"), ("logits", f"rand({batch_shape})")),
         funsor.Real,
+        xfail_reason="backend not supported" if get_backend() != "torch" else "",
```

Comment:
RelaxedBernoulli is a transformed distribution but currently, only "torch" backend supports TransformedDistribution.